frameworks/Rust/tokio-minihttp/benchmark_config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "framework": "tokio-minihttp",
+  "tests": [{
+    "default": {
+      "setup_file": "setup",
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "None",
+      "framework": "tokio-minihttp",
+      "language": "rust",
+      "orm": "raw",
+      "platform": "Rust",
+      "webserver": "tokio-minihttp",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "tokio-minihttp",
```

Comment:
Trailing commas are not valid in current JSON spec. This is breaking your `benchmark_config`. You can see a note about your benchmark_config in every failing test.